internal/oci/runtime_oci.go

```diff
@@ -630,11 +631,38 @@ func (r *runtimeOCI) DeleteContainer(c *Container) error {
 	return err
 }
 
+func updateContainerStatusFromExitFile(c *Container) error {
+	exitFilePath := filepath.Join(c.dir, "exit")
```

Comment:
I would add a function somewhere to get the exit file path and reuse that function in `UpdateContainerStatus()`